backend/src/main/resources/application-prod.yaml

```diff
@@ -1,6 +1,10 @@
 graphql.servlet.cors:
   allowed-origins:
     - https://simplereport.cdc.gov
+    - https://simple-report-api-prod.azurewebsites.net
+    - https://simple-report-prod.azureedge.net
+    - https://prod.simplereport.gov
+    - https://simplereport.gov
 simple-report:
   data-hub:
-    uploadurl: "https://prime-data-hub-prod.azurefd.net/api/reports"
+    uploadurl: "https://prime-data-hub-prod.azurefd.net/api/reports"
```

Comment:
This will probably cause a rebase with my PR #434. I'm ok doing that merge if you submit first.